src/targets/maven.ts

```diff
@@ -444,4 +391,105 @@ export class MavenTarget extends BaseTarget {
 
     return `${moduleName}.jar`;
   }
+
+  // Maven central does not indicate when it completes the action, so we need to
+  // retry every so often and query it for the new state of repository.
+  // Based on: https://github.com/vanniktech/gradle-maven-publish-plugin/ implementation.
+  public async closeAndReleaseRepository(): Promise<void> {
+    const repository = await this.getRepository();
+    const { repositoryId, type } = repository;
+
+    if (type !== 'open') {
+      throw new Error(
+        'No open repositories available. Go to Nexus Repository Manager to see what happened.'
+      );
+    }
+
+    await this.closeRepository(repositoryId);
+    await this.releaseRepository(repositoryId);
+  }
+
+  private getNexusRequestHeaders(): Record<string, string> {
+    return {
+      Accept: 'application/json',
+      Authorization: `Basic ${Buffer.from(
+        `${process.env.OSSRH_USERNAME}:${process.env.OSSRH_USERNAME}`
+      ).toString(`base64`)}`,
```

Comment:
Why parsing to base64?

Reply:
Basic Auth protocol is using base64 - https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#basic_authentication_scheme